recipes-kernel/linux/linux-mainline_5.3.8.bb

```diff
@@ -12,15 +12,17 @@ require linux.inc
 # in something or kernel-yocto.bbclass will fail.
 KBRANCH ?= "master"
 
+DEPENDS += "rsync-native"
```

Comment:
Why this line? Thx

Reply:
Build was failing lamenting missing rsync command.
Googling I found several references that some build steps now require rsync:
https://www.mail-archive.com/lfs-dev@lists.linuxfromscratch.org/msg04058.html